pingen/models/pingen_document.py

```diff
@@ -130,10 +130,10 @@ def _push_to_pingen(self, pingen=None):
                 % (self.id, pingen.api_url, e.description)
             )
             raise
-        except APIError:
+        except APIError as e:
             _logger.error(
-                "API Error when pushing Pingen Document %s to %s."
-                % (self.id, pingen.api_url)
+                "API Error when pushing Pingen Document %s to %s: %s"
+                % (self.id, pingen.api_url, str(e))
             )
```

Comment:
You should let the log handler interpolate strings
```suggestion
                "API Error when pushing Pingen Document %s to %s: %s", 
                self.id, pingen.api_url, str(e)
            )
```